internal/expr/expr_test.go

```diff
@@ -36,25 +36,23 @@ type District struct {
 type M map[string]any
 
 var tests = []struct {
-	summary        string
-	input          string
-	expectedParsed string
+	summary          string
+	input            string
+	expectedParsed   string
+	prepareArgs      []any
+	expectedPrepared string
 }{{
 	"star table as output",
 	"SELECT p.* AS &Person.*",
 	"[Bypass[SELECT ] Output[[p.*] [Person.*]]]",
-}, {
-	"multiple-quoted bypass expression",
-	`SELECT '''' AS &Person.*`,
-	`[Bypass[SELECT '''' AS ] Output[[] [Person.*]]]`,
-}, {
-	"single quote in double quotes",
-	`SELECT "'" AS &Person.*`,
-	`[Bypass[SELECT "'" AS ] Output[[] [Person.*]]]`,
+	[]any{Person{}},
+	"SELECT p.address_id AS [a-zA-Z_0-9]+, p.id AS [a-zA-Z_0-9]+, p.name AS [a-zA-Z_0-9]+",
```

Comment:
Given the other comment about simplifying the field name to `_sqlair_1` and so on, it would be nice to see those in the tests so we can both understand how it works here and also have logic that ensures our logic is right.

Reply:
Changed. They now don't use regex.